buildstockbatch/schemas/v0.3.yaml

```diff
@@ -36,6 +37,11 @@ gcs-spec:
 gcp-ar-spec:
   repository: str(required=True)
 
+gcp-job-environment:
+  vcpus: int(min=1, max=224, required=False)
```

Comment:
do we need encode/enforce the vcpus max? What if GCP offers new machines with more — I don't think the script should have to be updated to allow using those. What would happen if someone specifies a value higher than what's available?

Reply:
This is the amount of vCPU given to each task, not each machine. Then if the machines are large enough, they'll run multiple tasks at the same time.

This max is based on GCP batch allows. It's also way larger than anyone should really use for this job, so I feel okay with enforcing it here.